uniffi_bindgen/src/bindings/kotlin/templates/BooleanHelper.kt

```diff
@@ -1,15 +1,17 @@
-internal fun Boolean.Companion.lift(v: Byte): Boolean {
-    return v.toInt() != 0
-}
+internal object FfiConverterBoolean {
+    fun lift(v: Byte): Boolean {
+        return v.toInt() != 0
+    }
 
-internal fun Boolean.Companion.read(buf: ByteBuffer): Boolean {
-    return Boolean.lift(buf.get())
-}
+    fun read(buf: ByteBuffer): Boolean {
+        return lift(buf.get())
+    }
 
-internal fun Boolean.lower(): Byte {
-    return if (this) 1.toByte() else 0.toByte()
-}
+    fun lower(v: Boolean): Byte {
+        return if (v) 1.toByte() else 0.toByte()
+    }
 
-internal fun Boolean.write(buf: RustBufferBuilder) {
-    buf.putByte(this.lower())
-}
\ No newline at end of file
+    fun write(v: Boolean, buf: RustBufferBuilder) {
+        buf.putByte(lower(v))
+    }
+}
```

Comment:
I love that this refactor is moving converter methods away from the object itself.